Common/TableProducer/trackPropagation.cxx

```diff
@@ -214,6 +279,17 @@ struct TrackPropagation {
           tracksDCA(mDcaInfo[0], mDcaInfo[1]);
         }
       }
+
+      if (!fillPropagationHistograms) { // Fill histograms (if requested)
```

Comment:
Please invert the check and put the filling into the { ... } instead of the continue